example/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react'
+import { AppRegistry, StyleSheet, Text, View, TouchableOpacity, Platform } from 'react-native'
+
+import { GoogleSignin, GoogleSigninButton } from 'react-native-google-signin'
+import config from './config'
```

Comment:
the config file's name is currently config.example we need to change either the import or the name of the config file.